source/install/troubleshooting.rst

```diff
@@ -79,7 +79,25 @@ If email sign-in was turned off before the System Administrator switched sign-in
 
 4. Replace ``{username}`` with the name of the user you'd like to promote to an admin.
 
-SAML Issues
+Password resets
+~~~~~~~~~~~~~~~
+
+**Email address or username sign in**
+
+If you sign in to Mattermost using an email address or username, you can send a password reset request by selecting **I forgot my password**. 
+Mattermost Support do not have access to your Mattermost server/instance. If you experience any problems, the next step is to contact your Mattermost System Admin.
```

Comment:
@justinegeffen can you confirm if this should be "Mattermost Support does not...." Not sure of the correct way as when I read it out loud, I can see both working.

Reply:
Good point - I've reworded it. Let me know what you think? I think it reads better.